ipatests/test_integration/test_epn.py

```diff
@@ -655,6 +657,7 @@ def test_EPN_authenticated(self, cleanupmail):
             validate_mail(self.master, i,
                           "Hi test user,\n\nYour password will expire")
 
+    @skip_if_fips()
```

Comment:
I don't think that this skip is needed, the run from Nightly-STIG-Tier2/RHEL9.3/2023-09-15_00-42 shows a successful test.